prefect_dbt/cli/configs/postgres.py

```diff
@@ -0,0 +1,84 @@
+"""Module containing models for Postgres configs"""
+from typing import Any, Dict
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+from prefect_dbt.cli.configs.base import MissingExtrasRequireError, TargetConfigs
+
+try:
+    from prefect_sqlalchemy.credentials import DatabaseCredentials
+except ModuleNotFoundError as e:
+    raise MissingExtrasRequireError("Postgres") from e
+
+
+class PostgresTargetConfigs(TargetConfigs):
+    """
+    Target configs contain credentials and
+    settings, specific to Postgres.
+    To find valid keys, head to the [Postgres Profile](
+    https://docs.getdbt.com/reference/warehouse-profiles/postgres-profile)
+    page.
+
+    Args:
+        credentials: The credentials to use to authenticate; if there are
+            duplicate keys between credentials and TargetConfigs,
+            e.g. schema, an error will be raised.
+
+    Examples:
+        Instantiate PostgresTargetConfigs with DatabaseCredentials.
+        ```python
+        from prefect_dbt.cli.configs import PostgresTargetConfigs
+        from prefect_sqlalchemy import DatabaseCredentials, SyncDriver
+
+        credentials = DatabaseCredentials(
+            driver=SyncDriver.POSTGRESQL_PSYCOPG2,
+            username="prefect",
+            password="prefect_password",
+            database="postgres",
+            host="host",
+            port=8080
+        )
+        target_configs = PostgresTargetConfigs(credentials=credentials, schema="schema")
+        ```
+    """
+
+    _block_type_name = "dbt CLI Postgres Target Configs"
+    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/5zE9lxfzBHjw3tnEup4wWL/8cb73be51575a659667f6471a24153f5/dbt-bit_tm.png?h=250"  # noqa
+    _code_example = """/
+    ```python
+        from prefect_dbt.cli.configs import PostgresTargetConfigs
+        
+        dbt_cli_target_configs = PostgresTargetConfigs.load("BLOCK_NAME")
+    ```"""  # noqa
```

Comment:
Same deal with the description here.
```suggestion
    _block_type_name = "dbt CLI Postgres Target Configs"
    _logo_url = "https://images.ctfassets.net/gm98wzqotmnx/5zE9lxfzBHjw3tnEup4wWL/8cb73be51575a659667f6471a24153f5/dbt-bit_tm.png?h=250"  # noqa
    _code_example = """/
    ```python
        from prefect_dbt.cli.configs import PostgresTargetConfigs
        
        dbt_cli_target_configs = PostgresTargetConfigs.load("BLOCK_NAME")
    ```"""  # noqa
    _description = "dbt CLI target configs containing credentials and
    settings specific to Postgres."
```